src/components/cv/cv-innhold.css

```diff
@@ -0,0 +1,48 @@
+.cv-panel {
+    margin-left: 40px;
+    margin-right: 40px;
+    margin-top: 20px;
+    margin-bottom: 20px;
+    background-color: var(--a-gray-100);
+}
+
+.cv-container {
+    grid-row: row 2;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr;
```

Comment:
Går det ann å kjøre no repeat(6, 1fr)? bare for et litt kjappere oversikt

.. brukes dette (klassen cv-container)? (eller skal?)

Reply:
Ble ikke brukt tidligere nei, men endret til grid nå og endret litt på stylingen!